spec/models/finance/schedule/ecf_spec.rb

```diff
@@ -32,7 +32,7 @@
     end
   end
 
-  Cohort.where(start_year: 2021..).find_each do |cohort|
+  Cohort.where(start_year: 2021..2024).find_each do |cohort|
```

Comment:
Why did this need to change, of interest?

Reply:
We discussed this on call. The schedules.csv file creates up to 2024. We will change this after we add 2025 to the csv. I'll add a comment.